lib/net.c

```diff
@@ -775,10 +846,10 @@ check_port_offloads(struct gatekeeper_if *iface,
 	 *
 	 * Check each port for the RSS hash functions it supports,
 	 * and configure each to use the intersection of supported
-	 * hash functions out of GATEKEEPER_RSS_HF.
+	 * hash functions.
 	 */
 	iface->rss = true;
-	port_conf->rx_adv_conf.rss_conf.rss_hf = GATEKEEPER_RSS_HF;
+	port_conf->rx_adv_conf.rss_conf.rss_hf = 0;
```

Comment:
1. Let's add the following assert to the beginning of `check_port_offloads()` to keep up with DPDK: `RTE_BUILD_BUG_ON(GATEKEEPER_IPV4_RSS_HF | GATEKEEPER_IPV6_RSS_HF != ETH_RSS_IP);`

2. We should test `ipv4_if_configured()` and `ipv6_if_configured()` here and add `GATEKEEPER_IPV4_RSS_HF` and `GATEKEEPER_IPV6_RSS_HF` accordingly.